lib/Hakyll/Core/Compiler/Internal.hs

```diff
@@ -18,7 +18,7 @@ module Hakyll.Core.Compiler.Internal
     , compilerTell
     , compilerAsk
     , compilerThrow
-    , compilerFailMessage
+    , compilerNoResult
```

Comment:
I guess this one should also be re-exported from `Core.Compiler`, as it's useful for people writing custom `Context`s with error messages.